### PR TITLE
Centralize the initContainer that fixes ownership

### PR DESCRIPTION
**Pull request by yuvipanda**
We were copy-pasting this, and it was causing shared-readwrite
to not be functional in some cases.

Fixes https://github.com/2i2c-org/infrastructure/issues/440

### CODE REVIEW

**Comment by yuvipanda** [comment]
image

Nice

File: config/clusters/2i2c/utexas.values.yaml [hunk 1]
```diff
@@ -89,6 +89,26 @@ jupyterhub:
           # So we put data in a subpath
           subPath: data
     initContainers:
+      # Repeat this here even though it's defined in basehub/values.yaml - otherwise
+      # tthe other initContainer we define here will overwrite this
```
Comment (consideRatio) [inline config/clusters/2i2c/utexas.values.yaml:93]: This addition is to 2i2c/utexas --- should it be centralized also?

Comment (yuvipanda) [reply config/clusters/2i2c/utexas.values.yaml:93]: It needs to be repeated here, I left a comment explaining why there too - initContainers is a list so we can't just override it, as there is another initContainers here.

Comment (consideRatio) [reply config/clusters/2i2c/utexas.values.yaml:93]: 👍👍👍

**Review by consideRatio — APPROVED** [review]

**Review by damianavila — APPROVED** [review]
This is exactly what I had in mind when I talked about this one at the cycle meeting.
Thanks for investigating it and fixing it, @yuvipanda!!

**Comment by sgibson91** [comment]
I'll merge this one in since it has a few approvals

**Comment by damianavila** [comment]
@yuvipanda, is there anything else you want to add here, otherwise, I think it is ready...

**Comment by yuvipanda** [comment]
@damianavila ya just wanted to find a time when I've a few hours uninterrupted in case something goes wrong. Hopefully tomorrow.

**Pull request merged**